### PR TITLE
fix the issue that detected special commit incorrectly in use GenerateExternalContributors.ps1

### LINKED ISSUE #14724: GenerateExternalContributors.ps1 detected special commit incorrectly
## Description

Below 2 commits are considered as external contribution. But it comes from Azure PowerShell team. Please check what is different from other normal PR.

https://github.com/Azure/azure-powershell/commit/2bd57676fb986ca9e64e479656c9fa7357032138
https://github.com/Azure/azure-powershell/commit/8ec95a6ac3c13d681a93fe6a6e94ce3f05e5fa73

### PR DESCRIPTION
**Pull request by LucasYao93**
…eExternalContributors.ps1.

## Description

Fix #14724

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/GenerateExternalContributors.ps1 [hunk 1]
```diff
@@ -74,28 +73,46 @@ $sortPRs = $validPRs | Sort-Object -Property @{Expression = {$_.author.login}; D
...
-$contributorsMDHeaderFlag = $True
+# Get team members of the azure-powershell-team.
+$teamMembers = (Invoke-WebRequest -Uri "https://api.github.com/orgs/Azure/teams/azure-powershell-team/members" -Authentication Bearer -Token $token).Content | ConvertFrom-Json
+
+foreach ($members in $teamMembers) {
+  $skipContributors += $members.login
+}
+
+# Output exteneral contributors infomation.
+Write-Debug 'Output exteneral contributors infomation.'
```
Comment (dingmeng-xue) [inline tools/GenerateExternalContributors.ps1:84]:
```suggestion
Write-Debug 'Output external contributors information.'
```

**Review by dingmeng-xue — CHANGES_REQUESTED** [review]

File: tools/GenerateExternalContributors.ps1 (hunk 1 above)
Comment (LucasYao93) [reply tools/GenerateExternalContributors.ps1]: Done

**Review by wyunchi-ms — APPROVED** [review]

**Review by dingmeng-xue — DISMISSED** [review]

**Pull request merged**